src/main/scala/org/constellation/infrastructure/endpoints/CheckpointEndpoints.scala

```diff
@@ -83,6 +98,62 @@ class CheckpointEndpoints[F[_]](implicit F: Concurrent[F], C: ContextShift[F]) e
       } yield response
   }
 
+  private[endpoints] def postFinishedCheckpoint(
```

Comment:
There's a lot in common between this method and `SnapshotEndpoints.postSnapshotProposal` method. Maybe we could extract as much as we can into some service, and just send the `processFinishedCheckpointAsync` or `processProposalAsync` as a parameter?

Reply:
Done, it's unused and should be removed.

Reply:
I didn't extract more because I didn't expect using Gossip to something else. Now we see that we can abstract gossip even more and have common interface for gossip endpoints :+1: